pippo-controller-parent/pippo-controller/src/main/java/ro/pippo/controller/ControllerRegistry.java

```diff
@@ -195,11 +192,31 @@ private void registerControllerMethods(Map<Method, Class<? extends Annotation>>
                 // add an empty string to allow controllerPaths iteration
                 controllerPaths.add("");
             }
+            if (methodPaths.length == 0) {
```

Comment:
I see the same construction for `controllerPath`.  Is it not possible to remove these blocks and to make the code more readable?   For example `if (controllerPaths is empty) { // no controlers => return }`. The same for `methodPaths`.

Reply:
I could make this change but complexity would remain same. As user could define `@Path("")` or `@GET("")` in which case, controllerPath or methodPath would be non-empty.

I could covert the above using ternary operator
```java
controllerPaths = controllerPaths.isEmpty? Collections.singletonSet(""): controllerPaths
methodPaths = methodPaths.length == 0? new String[]{""}: methodPaths
```
does this look good or I misunderstood?